roles/syslog-aggregator/tasks/main.yml

```diff
@@ -24,7 +24,7 @@
   cron:
     name: log-cleanup
     user: root
-    job: "find {{ log_root }} -mtime +28 -type f -exec ls {} \\;"
+    job: "find {{ log_root }} -mtime +28 -type f -exec rm {} \\;"
```

Comment:
Using `;` with `-exec` can be very slow. Fortunately, deleting is so common there is a `-delete` option.